pkg/storage/scan.go

```diff
@@ -1,19 +1,57 @@
 package storage
 
 import (
-	"bytes"
 	"database/sql"
-	"encoding/json"
 	"fmt"
 	"github.com/kptm-tools/core-service/pkg/domain"
-	"time"
 )
 
-func (s *PostgreSQLStore) CreateScansTable() error {
+func (s *PostgreSQLStore) CreateScanTable() error {
 	query := `create table if not exists scans (
-      id UUID PRIMARY KEY,
-      status JSONB,
-      results JSONB,
+      id SERIAL PRIMARY KEY,
+      tenant_id UUID,
+      operator_id UUID,
+      started_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+      ended_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+  )`
+
+	_, err := s.db.Query(query)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+
+}
+
+func (s *PostgreSQLStore) CreateScanHostsTable() error {
+	query := `create table if not exists scans_hosts (
```

Comment:
`scan_hosts` should have a much simpler schema, this table is only for normalizing our db entities. 
```suggestion
query := `create table if not exists scan_hosts (
    id SERIAL PRIMARY KEY,
    scan_id INTEGER REFERENCES scans(id) ON DELETE CASCADE,
    host_id INTEGER REFERENCES hosts(id) ON DELETE CASCADE
)
```

Reply:
Ok Lucas, I will changed it